library/core/src/fmt/mod.rs

```diff
@@ -570,8 +570,23 @@ impl Display for Arguments<'_> {
 /// There are a number of helper methods on the [`Formatter`] struct to help you with manual
 /// implementations, such as [`debug_struct`].
 ///
-/// `Debug` implementations using either `derive` or the debug builder API
-/// on [`Formatter`] support pretty-printing using the alternate flag: `{:#?}`.
+/// For simple cases, it's also possible to implement `Debug` using the [`write!`] macro:
```

Comment:
Simple vs non-simple isn't necessarily the right distinction to draw here. Standard vs non-standard is probably more like it. In other words types that do not wish to use the standard suite of debug representations provided by the Formatter trait (debug_struct, debug_tuple, debut_list, debug_set, debug_map) can do something totally custom by manually writing an arbitrary representation to the Formatter.

Reply:
Made this change.